impl/src/bitfield_specifier.rs

```diff
@@ -85,31 +137,58 @@ fn generate_enum(input: syn::ItemEnum) -> syn::Result<TokenStream2> {
             // Use a raw identifier to allow strict keywords.
             Err(_) => format_ident!("r#{}", snake_variant),
         };
+        let value = if returns_result {
+            quote! { Ok(Self::#ident) }
```

Comment:
Please always use `quote_spanned!(span=> ...)` instead of `quote!{...}` with a proper span for the first `span` argument. This heavily improves the error messages during compilation.